.github/workflows/ci-java8.yaml

```diff
@@ -12,7 +12,8 @@ on:
       - '**' # should match all branches
 
 jobs:
-  ci_java_8:
+  ci_java8:
+    id: ci_java8
```

Comment:
I'd change the name to `ci_core_java8`, as this is to generate the package used in the other side and not a a *regular* CI process